polynote-frontend/polynote/ui.js

```diff
@@ -758,6 +761,12 @@ export class NotebookUI {
             }
         });
 
+        socket.addMessageListener(messages.UpdateCell, (path, globalVersion, localVersion, id, edits) => {
+            this.globalVersion = globalVersion;
+            // TODO: rebase the edits from localVersion to this.localVersion, and update the cell
```

Comment:
When we do this, it would be cool if we could highlight the change in some way - like when stuff happens somewhere else and propagates to our session it gets highlighted in green or something (which might fade away after a couple seconds).

Reply:
+1